t/02-with.t

```diff
@@ -10,20 +10,28 @@ use File::Temp ();
 use Inline (
     C           => 'DATA',
     with        => qw/Alien::OpenMP/,
-    directory   => (my $tmp = File::Temp::tempdir()),
+    directory   => ( my $tmp = File::Temp::tempdir() ),
     build_noisy => !!$ENV{HARNESS_IS_VERBOSE}
 );
 
 for my $num_threads (qw/1 2 4 8 16 32 64 128 256/) {
     is test($num_threads), $num_threads, qq{Ensuring compiled OpenMP program works as expected. Threads = $num_threads};
 }
 
+{
```

Comment:
Should this be a subtest with conditional skip on `$^O`?

Reply:
Sure, whatt should the condition be? I don't have a great handle on that.

Reply:
```suggestion
subtest "Inline config" => sub {
    plan skip_all => "skip OS is Windows" if $^O eq "MSWin32";
```

Reply:
I made it skip on if `Inline::C` is available.